frontend/src/components/main-view/details/supplier-details/accordion/list-accordion/ListAccordion.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+//component imports
+
+import SupplierListItem from "./supplier-list-item/SupplierListItem";
+import {List,} from "@mui/material";
+//interface imports
+import {IOrder} from "../../../../../../interfaces/IOrder";
+import {IProduct} from "../../../../../../interfaces/IProduct";
+
+type Props = {
+    items: IOrder[] | IProduct[]
+};
+
+function ListAccordion({items}: Props) {
+    const list = [...items].map((item) => <SupplierListItem key={item.id} item={item}/>);
```

Comment:
perhaps you could choose a more specific name here.

Reply:
this component is generic, taking unitl now either Product or Order as props. Therefor the list and its items are generic and not specific